docs/StardustDocs/d.tree

```diff
@@ -35,6 +35,7 @@
         <toc-element topic="DataColumn.md"/>
         <toc-element topic="DataRow.md"/>
     </toc-element>
+    <toc-element topic="nanAndNa.md"/>
```

Comment:
probably better to move under operations (for first time)

Reply:
But it's not an operation. I'm not comfortable having it there, seems weird. Probably we can find a better place after restructuring the docs/flattening them.